bundles/org.openhab.binding.comfoair/README.md

```diff
@@ -0,0 +1,889 @@
+| temperatures#indoorTemperatureIn   | Number:Temperature       | Indoor Temperature Incoming       | Inlet air temperature (inside)                                                              | no       | true      |
+| temperatures#indoorTemperatureOut  | Number:Temperature       | Indoor Temperature Outgoing       | Uptake air temperature (inside)                                                             | no       | true      |
+| temperatures#isT1Sensor            | Switch                   | Sensor T1 Available               | Availability of temperature sensor T1 (outdoor in)                                          | yes      | true      |
+| temperatures#isT2Sensor            | Switch                   | Sensor T2 Available               | Availability of temperature sensor T2 (indoor in)                                           | yes      | true      |
+| temperatures#isT3Sensor            | Switch                   | Sensor T3 Available               | Availability of temperature sensor T3 (indoor out)                                          | yes      | true      |
+| temperatures#isT4Sensor            | Switch                   | Sensor T4 Available               | Availability of temperature sensor T4 (outdoor out)                                         | yes      | true      |
+| temperatures#isEWTSensor           | Switch                   | EWT Sensor Available              | Availability of EWT temperature sensor                                                      | yes      | true      |
+| temperatures#isHeaterSensor        | Switch                   | Heater Sensor Available           | Availability of heater temperature sensor                                                   | yes      | true      |
+| temperatures#isCookerhoodSensor    | Switch                   | Cookerhood Sensor Available       | Availability of cookerhood temperature sensor                                               | yes      | true      |
+| temperatures#ewtTemperature        | Number:Temperature       | EWT Temperature                   | Temperature of geothermal heat exchanger sensor                                             | yes      | true      |
+| temperatures#heaterTemperature     | Number:Temperature       | Heater Temperature                | Temperature of heater sensor                                                                | yes      | true      |
+| temperatures#cookerhoodTemperature | Number:Temperature       | Cookerhood Temperature            | Temperature of cookerhood sensor                                                            | yes      | true      |
+| times#level0Time                   | Number                   | Level 0 duration                  | Operating hours at level 0 (away)                                                           | yes      | true      |
```

Comment:
`Operating hours` makes it sound like store hours.

Reply:
AFAIK "operating hours" is a common (technical) term for the time of operation of appliances. However, I'm open for suggestions.

Reply:
What about "up time"?

Reply:
Yeah, "uptime" sounds reasonable